openshift/templates/visual-verifier/visual-verifier-deploy.inoculation.test.param

```diff
@@ -5,6 +5,7 @@
 #=========================================================
 # NAME=visual-verifier
 # APP_NAME=visual-verifier
+# NAMESPACE_NAME=a99fd4
 # APP_GROUP=inoculation
 # SUFFIX=-inoculation
 # ROLE=visual-verifier
```

Comment:
This seems to be missing the new parameter.

Reply:
added param